src/components/Cards/NoResultsCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box, Heading, Text, useTheme } from '@chakra-ui/core';
+
+const NoResultsCard = ({ type }) => {
+  const theme = useTheme();
+
+  return (
+    <Box bg="rbb-light-gray" padding="1.5rem 2rem">
```

Comment:
For the padding here, you have access to the `theme` variable so you can grab our pre-defined spacing vars

`padding={[ theme.spacing.base, theme.spacing.lg ]}` Chakra takes an array of vars for top right bottom left when using shorthand :)